components/_components.scss

```diff
@@ -177,33 +177,6 @@
       padding: 1rem;
       font-size: 1.6rem;
     }
```

Comment:
Not many commits done. More commits could be used to show progress & easier to review / change if mistakes are made.